frontend/src/pages/modelCatalog/screens/RegisterCatalogModelForm.tsx

```
@@ -0,0 +1,210 @@
+        string_value: 'Model catalog',
+        metadataType: ModelRegistryMetadataType.STRING,
+      },
+      'Source model': {
+        // eslint-disable-next-line camelcase
+        string_value: model.name,
+        metadataType: ModelRegistryMetadataType.STRING,
+      },
+      'Source model version': {
+        // eslint-disable-next-line camelcase
+        string_value:
+          model.artifacts?.map((artifact) => artifact.tags && artifact.tags[0])[0] ?? '',
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)